GUI/XLSearchTaskWindow.xaml.cs

```diff
@@ -74,6 +74,11 @@ private void PopulateChoices()
                 ChildScanDissociationTypeComboBox.Items.Add(dissassociationType);
             }
 
+            foreach (string separationType in GlobalVariables.SeparationTypes)
```

Comment:
There is no need to add separationType for XL search.

Reply:
that part is used for the single+ peptides, not the xlink ones. defaults to lc